<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1374</width>
    <height>943</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>1211</width>
      <height>791</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>Backtest</string>
     </attribute>
     <widget class="QWidget" name="page1" native="true">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>10</y>
        <width>1181</width>
        <height>821</height>
       </rect>
      </property>
      <widget class="QTableWidget" name="valueTable">
       <property name="geometry">
        <rect>
         <x>600</x>
         <y>10</y>
         <width>531</width>
         <height>381</height>
        </rect>
       </property>
       <property name="alternatingRowColors">
        <bool>true</bool>
       </property>
       <property name="rowCount">
        <number>100</number>
       </property>
       <property name="columnCount">
        <number>7</number>
       </property>
       <row/>
       <row/>
       <row/>
       <row/>
       <row/>
       <row/>
       <row/>
       <row/>
       <row/>
       <row/>
       <row/>
       <row/>
       <row/>
       <row/>
       <row/>
       <row/>
       <row/>
       <row/>
       <row/>
       <row/>
       <row/>
       <row/>
       <row/>
       <row/>
       <row/>
       <row/>
       <row/>
       <row/>
       <row/>
       <row/>
       <row/>
       <row/>
       <row/>
       <row/>
       <row/>
       <row/>
       <row/>
       <row/>
       <row/>
       <row/>
       <row/>
       <row/>
       <row/>
       <row/>
       <row/>
       <row/>
       <row/>
       <row/>
       <row/>
       <row/>
       <row/>
       <row/>
       <row/>
       <row/>
       <row/>
       <row/>
       <row/>
       <row/>
       <row/>
       <row/>
       <row/>
       <row/>
       <row/>
       <row/>
       <row/>
       <row/>
       <row/>
       <row/>
       <row/>
       <row/>
       <row/>
       <row/>
       <row/>
       <row/>
       <row/>
       <row/>
       <row/>
       <row/>
       <row/>
       <row/>
       <row/>
       <row/>
       <row/>
       <row/>
       <row/>
       <row/>
       <row/>
       <row/>
       <row/>
       <row/>
       <row/>
       <row/>
       <row/>
       <row/>
       <row/>
       <row/>
       <row/>
       <row/>
       <row/>
       <row/>
       <column>
        <property name="text">
         <string>Order</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Value</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Cash</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Total Shares</string>
        </property>
       </column>
       <column/>
       <column/>
       <column/>
      </widget>
      <widget class="QGroupBox" name="autoStats">
       <property name="geometry">
        <rect>
         <x>600</x>
         <y>400</y>
         <width>441</width>
         <height>281</height>
        </rect>
       </property>
       <property name="title">
        <string>Auto Stats</string>
       </property>
       <widget class="QListWidget" name="statsOutput">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>30</y>
          <width>331</width>
          <height>231</height>
         </rect>
        </property>
       </widget>
       <widget class="QWidget" name="Backtest" native="true">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>270</y>
          <width>601</width>
          <height>191</height>
         </rect>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="Chart" native="true">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>10</y>
         <width>581</width>
         <height>491</height>
        </rect>
       </property>
      </widget>
      <widget class="QPlainTextEdit" name="debugOutput">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>510</y>
         <width>581</width>
         <height>171</height>
        </rect>
       </property>
       <property name="acceptDrops">
        <bool>false</bool>
       </property>
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>Live data</string>
     </attribute>
     <widget class="QWidget" name="liveData" native="true">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>941</width>
        <height>591</height>
       </rect>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1374</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
